internal/common/conversion/import_state.go

```diff
@@ -10,7 +10,26 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/resource"
 )
 
-func ImportStateProjectIDClusterName(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+type ClusterImportAttrNames struct {
+	ProjectID   string
+	ClusterName string
+}
+
+func (c *ClusterImportAttrNames) GetProjectID() string {
+	if c != nil && c.ProjectID != "" {
+		return c.ProjectID
+	}
+	return "project_id"
+}
+
+func (c *ClusterImportAttrNames) GetClusterName() string {
+	if c != nil && c.ClusterName != "" {
+		return c.ClusterName
+	}
+	return "cluster_name"
+}
+
+func ImportStateProjectIDClusterName(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse, names *ClusterImportAttrNames) {
```

Comment:
I could have made `names` an optional parameter with `names...` but felt passing `nil` is worth the extra effort instead of having an iterable argument

Reply:
changed in 49744fbe80fc1cebc33651fcf22c380e85cf6094